active-elastic-job.gemspec

```
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '>= 2.4.0', '< 2.7'
 
   spec.add_runtime_dependency 'aws-sdk-sqs', '~> 1.13'
-  spec.add_runtime_dependency 'rails',       '~> 5.2'
+  spec.add_runtime_dependency 'rails',       '=> 5.2'
```

That should be `>=` instead of `=>`. Also, we need to make sure that that version doesn't go to infinity. 

```suggestion
  spec.add_runtime_dependency 'rails',       '>= 5.2', '< 6.1'
```